neurodamus/io/sonata_config.py

```diff
@@ -380,17 +323,19 @@ def parsedStimuli(self):
             )
             stimulus["Mode"] = input_type_translation.get(stimulus["Mode"], stimulus["Mode"])
             stimuli[name] = stimulus
+
         return stimuli
 
     @property
     def parsedInjects(self):
         injects = {}
         # the order of stimulus injection could lead to minor difference on the results
         # so better to preserve it as in the config file
-        for name in self._sections["inputs"]:
+        for name in self._stable_inputs_order:
```

Comment:
Is `self._stable_inputs_order` going to be replace by the future `libsonata.inputs -> list` ? If so, please leave a TODO comment here.